tables/Tables/TableCache.cc

```diff
@@ -170,9 +170,8 @@ void TableCache::flushTable (const String& name,
 PlainTable* TableCache::lookCache (const String& name, int tableOption,
                                    const TableLock& lockOptions)
 {
-    ScopedMutexLock sc(itsMutex);
```

Comment:
or at least it appears to be the only state, you would assume that the synchronization of the calls into the shared table object would be handled by the lock of he table.
But it doesn't seem to have a lock so maybe we need to make it recursive ..